tests/testthat/test_barber_smooth.R

```diff
@@ -0,0 +1,12 @@
+test_that("The function runs", {
+  vec <- c(1, 7, 5, 6, 19, 10)
+  smoothed <- barber_smooth(vec, n_neighbors = 2, times = 1)
+
+  expect_equal(length(smoothed), 6)
+  expect_equal(class(smoothed), "matrix")
+  expect_equal(
+    smoothed,
+    matrix(data = c(2.79, 3.81, 5.25, 8.32, 8.69, 10.93)),
```

Comment:
I literally just pasted the actual results to create this test data. I assume this is reasonable. Please give it a quick glance

Reply:
yes. this is very reasonable.